hilfe/handbuch/mitarbeiter/arbeitszeitkonto/index.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: layouts/hilfe/single.njk
+title: Arbeitszeitkonto
+weight: 2
+icon: scale-unbalanced
+---
+
+Im Arbeitszeitkonto werden die vertraglich vereinbarten, den tatsächlich geleisteten Arbeitszeiten gegenübergestellt und Über- oder Minderstunden ausgewiesen.
+
+Die Gegenüberstellung erfolgt nur auf Basis der zum aktuellen Zeitpunkt erfassten Arbeitszeiten. **Geplante Arbeitszeiten werden im Arbeitszeitkonto nicht berücksichtigt**.
+
+Unter [Konten (Übersicht)](/hilfe/handbuch/mitarbeiter/konten/) wird das Arbeitszeitkonto gemeinsam mit dem Zuschlagskonto und Urlaubskonto in einer gemeinsamen Übersicht zusammengefasst. 
+
+Sofern nicht anders erfasst, werden die Über- oder Minderstunden am Ende eines Jahres immer automatisch in das Folgejahr übertragen. Wie Sie diese Überträge bearbeiten können, erfahren Sie unter dem Punkt Konten Zurücksetzen.
+
+## Übersicht
+
+In der Übersicht können Sie die Arbeitszeiten all ihrer Mitarbeiter über einen selbst gewählten Zeitraum einsehen. Dabei werden fünf Werte im Format Stunden:Minuten angezeigt:
+
+*Übertrag -* sind die Über- oder Minderstunden, welche zu Beginn des ausgewählten Zeitraums vom Mitarbeiter angesammelt sind. 
+
+*Soll -* sind die über die [Vertragsbedingungen](/hilfe/handbuch/mitarbeiter/vertrag/) kalkulierten Soll-Stunden des Mitarbeiters für den Zeitraum. 
+
+*Ist -* sind die tatsächlich erarbeiteten Stunden des Mitarbeiters.
+
+*Differenz -* sind die Ist-Stunden abzüglich der Soll-Stunden - dieser Wert gibt an ob Planung und Durchführung von Arbeitszeiten sich decken. 
+
+*Saldo -* sind die insgesamt angesammelten Über- und Minderstunden.
```

Comment:
das "-" ist hier fast unsichtbar durch den Farbverlauf, vielleicht rausnehmen und in schwarz oder ":"?

Reply:
Ist in dem Fall uniform mit z.B dem Einstellungs-Artikeln. Ich finde den Farbverlauf stylistisch schön aber wenn es schwer lesbar ist, kann ich auch eine andere Norm für Aufzählungen verwenden.